tests/Composer/Test/Util/ZipTest.php

```diff
@@ -99,22 +101,9 @@ public function testReturnsComposerJsonInFirstFolder()
         }
 
         $result = Zip::getComposerJson(__DIR__.'/Fixtures/Zip/folder.zip');
-
         $this->assertEquals("{\n    \"name\": \"foo/bar\"\n}\n", $result);
     }
 
-    public function testMultipleTopLevelDirsIsInvalid()
-    {
-        if (!extension_loaded('zip')) {
-            $this->markTestSkipped('The PHP zip extension is not loaded.');
-            return;
-        }
-
-        $result = Zip::getComposerJson(__DIR__.'/Fixtures/Zip/multiple.zip');
-
-        $this->assertNull($result);
-    }
```

Comment:
Not sure why this test is gone. Seems like it should also be throwing now. Also there is no test referencing your new multiple_subfolders.zip anymore it seems.

Reply:
sorry about that, added it back and added my test as well